lib/front_end.rb

```diff
@@ -3,7 +3,7 @@ def self.host
     if Rails.env.production?
       'https://www.zooniverse.org'
     else
-      'http://demo.zooniverse.org/panoptes-front-end'
+      'https://pfe-preview.zooniverse.org'
```

Comment:
Should we make this an configurable ENV var with this as the default?
`ENV.fetch("FRONT_END_URL", 'https://pfe-preview.zooniverse.org')`